pkg/services/evaluation_report/evaluation_report_deleter_test.go

```diff
@@ -0,0 +1,70 @@
+package evaluationreport
+
+import (
+	"github.com/gofrs/uuid"
+
+	"github.com/transcom/mymove/pkg/appcontext"
+	"github.com/transcom/mymove/pkg/apperror"
+	"github.com/transcom/mymove/pkg/auth"
+	"github.com/transcom/mymove/pkg/models"
+	"github.com/transcom/mymove/pkg/services"
+	"github.com/transcom/mymove/pkg/testdatagen"
+)
+
+func (suite *EvaluationReportSuite) TestEvaluationReportDeleter() {
+	setupTestData := func() (services.EvaluationReportDeleter, models.EvaluationReport, appcontext.AppContext) {
+		deleter := NewEvaluationReportDeleter()
+		move := testdatagen.MakeDefaultMove(suite.DB())
+		shipment := testdatagen.MakeMTOShipment(suite.DB(), testdatagen.Assertions{
+			MTOShipment: models.MTOShipment{
+				MoveTaskOrderID: move.ID,
+			},
+		})
+		officeUser := testdatagen.MakeOfficeUser(suite.DB(), testdatagen.Assertions{})
+		report := testdatagen.MakeEvaluationReport(suite.DB(), testdatagen.Assertions{
+			EvaluationReport: models.EvaluationReport{
+				ShipmentID: &shipment.ID,
+				Type:       models.EvaluationReportTypeShipment, OfficeUserID: officeUser.ID,
+			},
```

Comment:
This is another case where the testdatagen function is intended to work with flattened assertions. I think your code will work just fine, but it will create an extra office user and move behind the scenes.

```suggestion
			OfficeUser: officeUser,
			MTOShipment: shipment,
			Move: move,
```

Reply:
Another great suggestion, fixed ✅